v0.3/cruorg/components/list/list.scss

```diff
@@ -2,3 +2,18 @@
 @import '../../branding-imports';
 
 /* List styles */
+
+@mixin cru-list-extra-spacing {
+    margin-bottom: $cru-spacer;
+}
+
+.list {
+    .cmp-list__item {
+        @include cru-text;
+        font-size: $cru-font-size-base;
```

Comment:
@sebamelzi You can lose this `font-size` rule as it's already being defined as part of the `cru-text` mixin. Yes, it'll be a little bigger than what's in the design but I think that was an oversight by the designers.